projects/online/run.sh

```diff
@@ -16,13 +17,16 @@ export ONLINE_DATADIR=/dev/shm/kafka/
 # where results and deployment logs will be writen
 export AFRAME_ONLINE_OUTDIR=
 
+# where aframe containers are stored
+export AFRAME_CONTAINER_ROOT=
+
 config=/path/to/config.yaml
 
 export CUDA_VISIBLE_DEVICES=
 crash_count=0
 until apptainer run --nv --bind $XDG_RUNTIME_DIR:$XDG_RUNTIME_DIR --env CUDA_VISIBLE_DEVICES=$CUDA_VISIBLE_DEVICES --env AFRAME_ONLINE_OUTDIR=$AFRAME_ONLINE_OUTDIR --env ONLINE_DATADIR=$ONLINE_DATADIR --env AFRAME_WEIGHTS=$AFRAME_WEIGHTS --env AMPLFI_WEIGHTS=$AMPLFI_WEIGHTS $AFRAME_CONTAINER_ROOT/online.sif /opt/env/bin/online --config $config 2>> monitoring.log; do
```

Comment:
I missed this PR before. I think it just needs to specify `AMPLFI_HL_WEIGHTS` and `AMPLFI_HLV_WEIGHTS` in the command here.